src/super_gradients/training/models/segmentation_models/ppliteseg.py

```diff
@@ -0,0 +1,383 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from typing import Union, List, Tuple
+from super_gradients.training.utils.module_utils import ConvBNReLU, UpsampleMode, make_upsample_module
+from super_gradients.training.models.segmentation_models.stdc import SegmentationHead, AbstractSTDCBackbone, STDC1Backbone, STDC2Backbone
+from super_gradients.training.models.segmentation_models.segmentation_module import SegmentationModule
+from super_gradients.training.utils import HpmStruct, get_param
+
+
+class SPPM(nn.Module):
+    """
+    Simple Pyramid Pooling context Module.
```

Comment:
It is actually a design of the ppliteseg paper authors, they are similar in a way of applying pooling layers with different kernel sizes. The differences are that here average pooling is used instead of max pooling and the branches are summed instead concatenation.